Used by dwm-5.9-uselessgap.diff for tile/bstack layout to work correctly.

--- a/dwm.c	2011-12-19 16:02:46.000000000 +0100
+++ b/dwm.c	2013-09-11 16:16:38.777607582 +0200
@@ -1718,12 +1718,12 @@
 		if(i < m->nmaster) {
 			h = (m->wh - my) / (MIN(n, m->nmaster) - i);
 			resize(c, m->wx, m->wy + my, mw - (2*c->bw), h - (2*c->bw), False);
-			my += HEIGHT(c);
+			my += HEIGHT(c) + 2*gappx;
 		}
 		else {
 			h = (m->wh - ty) / (n - i);
 			resize(c, m->wx + mw, m->wy + ty, m->ww - mw - (2*c->bw), h - (2*c->bw), False);
-			ty += HEIGHT(c);
+			ty += HEIGHT(c) + 2*gappx;
 		}
 }
 
